part-ii-domain-logic/deadlines.md

```diff
@@ -0,0 +1,74 @@
+# Deadlines
+
+The 'Deadline concept' in the Axon Framework is a mechanism which enables certain actions (in our case a `@DeadlineHandler` annotated method) to be executed after certain amount of time. The context of this execution is an Aggregate or a Saga in which the Deadline was scheduled. If the Deadline becomes obsolete there is the possibility to cancel it as well.  
+
+Deadlines are possible to be scheduled from Sagas and Aggregates. The `DeadlineManager` component is responsible for scheduling deadlines and invoking `@DeadlineHandler`s when the deadline is met. The `DeadlineManager` can be injected as a resource. It has two flavors: `SimpleDeadlineManager` and `QuartzDeadlineManager`, just like the [Event Scheduling](sagas.md#keeping-track-of-deadlines) mechanism for Sagas. 
+
+## Scheduling a Deadline
+
+A deadline can be scheduled only by providing a `Duration` after which it will be triggered (or `Instance` at which it will be triggered) and a name.
+
+> **Note**
+>  
+> Unlike [Event Scheduling](sagas.md#keeping-track-of-deadlines), when a Deadline is triggered there will be no storing of the published Message.
```

Comment:
I would explain that it doesn't involve the EventBus. That means, when using an Event Store, the deadline messages are not stored.